apps/elixir_ls_debugger/lib/debugger/server.ex

```diff
@@ -865,6 +871,35 @@ defmodule ElixirLS.Debugger.Server do
         do: save_and_reload(module, beam_bin)
   end
 
+  defp interpret_modules_patterns(file_patterns) do
+    regexes =
+      Enum.flat_map(file_patterns, fn pattern ->
+        case Regex.compile(pattern) do
+          {:ok, regex} ->
+            [regex]
+
+          {:error, error} ->
+            IO.puts(
+              :standard_error,
+              "Unable to compile file pattern (#{inspect(pattern)}) into a regex. Received error: #{inspect(error)}"
+            )
+
+            []
+        end
+      end)
+
+    ElixirSense.all_modules()
+    |> Enum.filter(fn module_name ->
+      Enum.find(regexes, fn regex ->
+        Regex.match?(regex, module_name)
+      end)
+    end)
+    |> Enum.map(fn module_name -> Module.concat(Elixir, module_name) end)
+    |> Enum.each(fn module ->
+      :int.ni(module)
```

Comment:
Add try catch around ni call

Reply:
Added in 5b2bc59ecc88179328cf21ea3df0ba37b434938e